core/src/main/java/org/everit/json/schema/loader/SchemaLoader.java

```diff
@@ -232,11 +237,8 @@ public SchemaLoader(final SchemaLoaderBuilder builder) {
     }
 
     private EnumSchema.Builder buildEnumSchema() {
-        Set<Object> possibleValues = new HashSet<>();
         JSONArray arr = ls.schemaJson.getJSONArray("enum");
-        IntStream.range(0, arr.length())
-                .mapToObj(arr::get)
-                .forEach(possibleValues::add);
+        Set<Object> possibleValues = Sets.newHashSet(arr);
```

Comment:
... okay maybe not, it implements the Iterable in newer versions, but I'm not sure which version does the project use